.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+name: Unit Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
@PeterNgTr I suggest to use a fixed ubuntu version here

Reply:
it is by default of github actions and being used here -> https://github.com/Codeception/CodeceptJS/blob/master/.github/workflows/check.yml#L5

do you have suggestion which version should it be?

Reply:
@PeterNgTr I suggest to use current latest explicitely, it would be `ubuntu-18.04`